### PR TITLE
Update line break table per Unicode 14.0

### PR DESCRIPTION
**Pull request by makotokato**
Fix https://github.com/unicode-org/icu4x/issues/1122. Unicode 14.0 data requires emoji-data.txt for `Extended_Pictographic`.

### CODE REVIEW

File: experimental/segmenter/tools/generate_properties.py [hunk 1]
```diff
@@ -43,19 +48,45 @@
...
+
+        if prop != "Extended_Pictographic":
+            continue
+
+        # We have a success match, property is Extended_Pictographic, and the
+        # codepoints are in plane 0 & 1. Store their Extended_Pictographic
+        # list.
+        if prop and start < begin_plane2:
+            for i in range(start, end + 1):
+                extended_pictographics[i] = "Y"
```
Comment (aethanyc) [inline experimental/segmenter/tools/generate_properties.py:73]: Question: could be elaborate a bit more on using "Y"? Does it mean "yes" or something else?

File: experimental/segmenter/tools/generate_properties.py [hunk 2]
```diff
@@ -73,6 +104,12 @@
                     # codepoints 0x0029 and 0x005D with CP line break class both
                     # have EA property "Na".
                     lb_props[i] = "CP_EA"
+                elif uni_prop == "Cn":
+                    # For LB30b
+                    if prop == "ID" and extended_pictographics[i] == "Y":
```
Comment (aethanyc) [inline experimental/segmenter/tools/generate_properties.py:109]: Nit: Maybe quote the spec here "Unassigned codepoints with Line_Break=ID in some blocks are also assigned the Extended_Pictographic property." 

http://www.unicode.org/reports/tr51/#def_level1_emoji

**Review by aethanyc — COMMENTED** [review]
Thank you for working on this. This looks good to me except some minor suggestions.

Suggestion: While we are using python to generate tables, could you add a link to `emoji-data.txt` in [lib.rs](https://github.com/unicode-org/icu4x/blob/7769518dc099f1763061aa3be3df86bacffd9cbb/experimental/segmenter/src/lib.rs#L84), and regenerate README? 

I believe the link is 
```
https://www.unicode.org/Public/UCD/latest/ucd/emoji/emoji-data.txt
```

File: experimental/segmenter/tools/generate_properties.py [hunk 3]
```diff
@@ -43,19 +48,45 @@
             for i in range(start, end + 1):
                 ea_props[i] = prop
 
+with open('emoji-data.txt', 'r') as emoji_file:
```
[the remark this answers is not in the document]
Comment (makotokato) [reply experimental/segmenter/tools/generate_properties.py:51]: Yes.

File: experimental/segmenter/tools/generate_properties.py (hunk 1 above)
Comment (makotokato) [reply experimental/segmenter/tools/generate_properties.py:73]: I will change it to boolean.

File: experimental/segmenter/tools/generate_properties.py [hunk 4]
```diff
@@ -20,6 +20,11 @@
 # http://www.unicode.org/reports/tr11/#ED7
 ea_props = ["N" for x in range(begin_plane2)]
 
+# Set default value for Extended Pictographic to "N" (No) for all codepoints
+# Asian) for all the codepoints in Unicode Plane 0 and PlaneA 1
```
[the remark this answers is not in the document]
Comment (makotokato) [reply experimental/segmenter/tools/generate_properties.py:24]: Ah, this is typo

**Review by aethanyc — APPROVED** [review]

**Pull request merged**